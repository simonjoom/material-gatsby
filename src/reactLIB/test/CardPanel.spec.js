import React from 'react';
import { shallow } from 'enzyme';
import CardPanel from '../src/CardPanel';

const modifier = 'blue-grey';

describe('<CardPanel />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <CardPanel className={modifier}>I am a very simple card</CardPanel>
    );
  });

  test('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render', () => {
    expect(wrapper.find('.card-panel')).toHaveLength(1);
  });

  test('accepts className prop', () => {
    expect(wrapper.hasClass(modifier)).toBeTruthy();
  });
});
