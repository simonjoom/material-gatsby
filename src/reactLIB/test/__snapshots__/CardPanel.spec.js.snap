// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardPanel /> renders 1`] = `
<div
  className="card-panel blue-grey"
>
  I am a very simple card
</div>
`;
